src/main.cpp

```diff
@@ -49,6 +49,8 @@ constexpr unsigned long update_buzzer_controller_period_us = 100000; // 100 000
 constexpr unsigned long kick_watchdog_period_us = 10000;             // 10 000 us = 100 Hz
 constexpr unsigned long ams_update_period_us = 10000;                // 10 000 us = 100 Hz
 constexpr unsigned long ethernet_update_period = 10000;
+constexpr unsigned long ioexpander_sample_period_us = 250;
```

Comment:
We do not need to sample this at 250 microseconds. That's 4000Hz, and these signals are not changing quickly. Set this to something like 50,000 microseconds (20Hz). That should be sufficiently fast.